docker/uwsgi/index.html

```diff
@@ -1,73 +1,94 @@
+      border-color: #707070;
+      color: #fff;
+    }
+    p {
+      font-size: 14px;
+      margin-bottom: 14px;
+    }
+    h5 {
+      margin-bottom: 5px;
+    }
+    h3 {
+      color: #50596C;
+    }
+    img {
+      margin-top: 10px;
+    }
+  </style>
 </head>
 <body>
 
-<div class="container" id="content">
+<div class="container" style="margin-top: 50px;" id="content">
   <div class="columns">
-    <div class="column col-6 col-mx-auto">
-        <h1>µRaiden demo</h1>
+    <div class="column col-9 col-mx-auto">
+        <h1>µRaiden Demos</h1>
     </div>
   </div>
-
   <div class="columns">
-  <div class="column col-6 col-mx-auto" id="description">
+  <div style="margin-top: 10px" class="column col-9 col-mx-auto" id="description">
     <h5>1. Install Web3 browser extension</h5>
-    <p>This allows you to access the Ethereum networks. Use <a href="https://metamask.io/">Metamask</a> or <a href="https://chrome.google.com/webstore/detail/parity-ethereum-integrati/himekenlppkgeaoeddcliojfddemadig">Parity</a> - works in Chrome and Firefox</p>
-    <h5>2. Get some Ropsten Testnet Ether (For free!)</h5>
-    <p>Try <a href="http://faucet.ropsten.be:3001/">some</a> <a href="https://faucet.metamask.io/">faucets</a></p>
-    <h5>3. Buy µRaiden demo tokens with your ether</h5>
-    <p>You can buy some when you first access the resource.</p>
-    <h5>4. Access the content</h5>
-    <p>And thanks for coming!</p>
+    <p><a style="color: #a3a3a3" href="https://metamask.io/">Metamask</a> and <a style="color: #a3a3a3" href="https://chrome.google.com/webstore/detail/parity-ethereum-integrati/himekenlppkgeaoeddcliojfddemadig">Parity</a> allow you to access the Ethereum DApps, in both Chrome and Firefox.</p>
+    <h5>2. Get some Kovan Testnet Ether</h5>
+    <p>Paste your ETH address in the <a style="color: #a3a3a3" href="https://gitter.im/kovan-testnet/faucet">Kovan gitter</a> or try <a style="color: #a3a3a3" href="https://github.com/kovan-testnet/faucet">other faucets.</a></p>
+    <h5>3. Enter the demo</h5>
+    <p>Open Metamask on the Ethereum Kovan testnet and test µRaiden.</p>
   </div>
   </div>
-
-
   <div class="columns">
-    <div class="column col-4 col-ml-auto">
-      <div class="empty">
-        <div class="empty-icon">
-          <i class="icon icon-people"></i>
-        </div>
-        <p class="empty-title h5">Buy fortune cookie</p>
-        <p class="empty-subtitle">(words of wisdom, English)</p>
-        <div class="empty-action">
-          <a class="btn btn-primary" id="fortunes_en" href="/fortunes_en">Buy (1 TKN)</a>
-        </div>
+    <div class="column col-5 col-mr-auto col-ml-auto">
+      <div class="empty" style="padding: 32px">
+        <h3>Pay-per-View</h3>
+        <p class="empty-subtitle">Access online content through micropayments</p>
+        <a id="wiki" href="https://wikidemo.micro.raiden.network/wiki/">
+          <img class="grayscale" src="../static/wikiscreen.jpg" width="100%" height="auto">
```

Comment:
Please, don't use `../` relative path, or it'll break on nginx. Use simply `/static/wikiscreen.jpg`, and I'll make a route in nginx for this path.